res/values/strings.xml

```diff
@@ -1462,45 +1449,45 @@
     <string name="preferences__enable_if_your_device_supports_sms_mms_delivery_over_wifi">Enable if your device uses SMS/MMS delivery over WiFi (only enable when \'WiFi Calling\' is enabled on your device)</string>
     <string name="preferences__incognito_keyboard">Incognito keyboard</string>
     <string name="preferences__read_receipts">Read receipts</string>
-    <string name="preferences__if_read_receipts_are_disabled_you_wont_be_able_to_see_read_receipts">If read receipts are disabled, you won\'t be able to see read receipts from others.</string>
+    <string name="preferences__if_read_receipts_are_disabled_you_wont_be_able_to_see_read_receipts">If read receipts are disabled, others won\'t be able to see if you read their messages, but you also won\'t be able to see if others have read your messages.</string>
     <string name="preferences__typing_indicators">Typing indicators</string>
-    <string name="preferences__if_typing_indicators_are_disabled_you_wont_be_able_to_see_typing_indicators">If typing indicators are disabled, you won\'t be able to see typing indicators from others.</string>
+    <string name="preferences__if_typing_indicators_are_disabled_you_wont_be_able_to_see_typing_indicators">If typing indicators are disabled, others won\'t be able to see when you are typing, but you also won\'t be able to see when others are typing.</string>
```

Comment:
A full sentence reads easier and can be cited to friends. Yes it can be made shorter and that will be fine too, but why? These aren't too long.

Reply:
Some languages does not support compound nouns and they need longer constructions to refer to "typing indicators" as subject. But ok, it can be fixed in those translations.